app/controllers/browse_controller.rb

```diff
@@ -45,7 +45,7 @@ def self.modified_field
     #   The ordering of the field names is the order of the display
     config.add_facet_field solr_name("resource_type", :facetable), label: "Item Type", limit: 20
     config.add_facet_field solr_name("subject", :facetable), label: "Subject", limit: 20
-    config.add_facet_field solr_name("hasCollection", :symbol), label: "Collection", limit: 20 
+    #config.add_facet_field solr_name("hasCollection", :symbol), label: "Collection", limit: 20 
```

Comment:
@nnunn has temporarily disabled the browsing by collection with @sfbetz's request. We will need sometime to sort out browsing by collection/community after the launch. But I'm okay with removing the line.